doc/faq.md

```diff
@@ -9,27 +9,36 @@ Configuration can be put in either *xxx.cli.md* or *xxx.az.md* in *Azure/azure-r
 
 
 ## How to install the codegen extension from the swagger pipeline
-* Before the Azure CLI next release (2020.04.21). You need to build azure-cli with the dev branch which contains the new azure-core. So that the codegen extension can work.
-* The steps are
+* Please install the latest Azure CLI and try this.
 ``` yaml
-git clone git@github.com:Azure/azure-cli.git
-git clone git@github.com:Azure/azure-cli-extensions.git
-cd azure-cli
-git checkout dev
-# you need to get your python3 as python ready.
-python -m virtualenv env
-source env/bin/activate
+az extension add --source=<cli-extension-whl-link-from-swagger-pipeline>
+```
+
+
+## How to install the codegen extension from the generated code  
+There're two ways to install the extension
+* install the extension in a dev mode.  
+```yaml
 pip install azdev
-azdev setup -c -r ../azure-cli-extensions/
+azdev extension add <extension-name>
+```
+* build the extension and install the extension in a non-dev mode
+``` yaml
+# how to build the generated file into a wheel file
 
-# after the azdev setup you get the az
+# 1. use python original way
+python setup.py sdist bdist_wheel
 
-# When the Azure CLI with new azure core feature released. you can skip all the step above. just run 
-pip install azure-cli 
+# 3. use azdev to build the wheel file
```

Comment:
2? and user only needs to pick 1 or 3, right? then it's better to put them in one bullet, like
1. build whl file, there are two ways to do it
    a....
    b....

Reply:
Yes but user can also choose not to build the wheel file

Reply:
Changed. Please review again. thx